furnace.c

```diff
@@ -207,6 +226,10 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
 #endif
 #if CONFIG_MAGNETRON
                         "pulse <0:127>     \t\t starts pulses of magnetron\n"
+#endif
+#if CONFIG_WATER
+                        "water <0:10>      \t\t sets pwm duty of water channel\n"
```

Comment:
I can't see the user input sanitization when we parse `water %u`. We should enforce the value to be within this range.